### DISCUSSION TITLE
Power operator is replaced with Math.pow

### DISCUSSION THREAD

**Discussion by mesudip** [webpack/webpack#14390] [Q&A] [answered]
**What is the current behavior?**
I got weird error for a statement that works when copy pasting to browser console : `Can't convert BigInt to number`

It's because, the statement
```javascript
let rate=BigInt(this.sale.rate) * (BigInt(10)** BigInt(this.decimals))
```
was complied to 
```javascript
var rate = BigInt(this.sale.rate) * Math.pow(BigInt(10), BigInt(this.decimals))
```

**Other relevant information:**
webpack version:   @nuxt/webpack 2.16.8 which uses webpack 4.46.0
Node.js version:  v14.16.1
Operating System: -
Additional tools: -

**Comment by alexander-akait** [comment, marked as answer]
Do you use `babel`?

**Comment by mesudip** [comment]
I am using Nuxt, It's basically Vue.js, I don't know what it uses inside, but I am not using typescript.

**Reply by alexander-akait** [reply]
I think nuxtjs uses babel under the hood, so babel rewrite ** on Math.pow(), please ask nuxt team how to allow new syntax

**Reply by mesudip** [reply]
I was writing  issue and mistakenly posted midway, I think this should be moved to issue.

**Reply by mesudip** [reply]
Ok I'll do that.
